### PR TITLE
changelog for v2.13.0

### PR DESCRIPTION
**Pull request by kdomanski**
```release-note
NONE
```

/assign @xrstf 
/assign @zreigz 
/hold

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,115 @@
...
+- Fixed broken NodePort-Proxy for user clusters with LoadBalancer expose strategy. [#4590](https://github.com/kubermatic/kubermatic/issues/4590) ([xrstf](https://github.com/xrstf))
+- Fixed cluster namespaces being stuck in Terminating state when deleting a cluster. [#4619](https://github.com/kubermatic/kubermatic/issues/4619) ([xrstf](https://github.com/xrstf))
+- Fixed Seed Validation Webhook rejecting new Seeds in certain situations [#4662](https://github.com/kubermatic/kubermatic/issues/4662) ([xrstf](https://github.com/xrstf))
+- A panic that could occur on clusters that lack both credentials and a credentialsSecret was fixed. [#4742](https://github.com/kubermatic/kubermatic/issues/4742) ([alvaroaleman](https://github.com/alvaroaleman))
+- A bug that occasionally resulted in a `Error: no matches for kind &#34;MachineDeployment&#34; in version &#34;cluster.k8s.io/v1alpha1&#34; visible in the UI was fixed. [#4870](https://github.com/kubermatic/kubermatic/issues/4870) ([alvaroaleman](https://github.com/alvaroaleman))
```
Comment (zreigz) [inline CHANGELOG.md:45]: not ended quotation mark

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+- Added RBAC management functionality to UI [#1815](https://github.com/kubermatic/dashboard-v2/issues/1815) ([kgroschoff](https://github.com/kgroschoff))
+
+**Cloud providers:**
+- Openstack: A bug that caused cluster reconciliation to fail if the controller crashed at the wrong time was fixed [#4754](https://github.com/kubermatic/kubermatic/issues/4754) ([alvaroaleman](https://github.com/alvaroaleman))
+- Openstack: New Kubernetes 1.16&#43; clusters use the external Cloud Controller Manager and CSI by default [#4756](https://github.com/kubermatic/kubermatic/issues/4756) ([alvaroaleman](https://github.com/alvaroaleman))
+- vSphere: Fixed a bug that resulted in a faulty cloud config when using a non-default port [#4562](https://github.com/kubermatic/kubermatic/issues/4562) ([alvaroaleman](https://github.com/alvaroaleman))
+- vSphere: fixed a bug which cased custom VM folder paths not to be put in cloud-configs [#4737](https://github.com/kubermatic/kubermatic/issues/4737) ([kdomanski](https://github.com/kdomanski))
```
Comment (xrstf) [inline CHANGELOG.md:31]: Either we have both vSphere bugfixes with an uppercase "Fixed" or not ;-)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,115 @@
...
+- Fixed Seed Validation Webhook rejecting new Seeds in certain situations [#4662](https://github.com/kubermatic/kubermatic/issues/4662) ([xrstf](https://github.com/xrstf))
+- A panic that could occur on clusters that lack both credentials and a credentialsSecret was fixed. [#4742](https://github.com/kubermatic/kubermatic/issues/4742) ([alvaroaleman](https://github.com/alvaroaleman))
+- A bug that occasionally resulted in a `Error: no matches for kind &#34;MachineDeployment&#34; in version &#34;cluster.k8s.io/v1alpha1&#34; visible in the UI was fixed. [#4870](https://github.com/kubermatic/kubermatic/issues/4870) ([alvaroaleman](https://github.com/alvaroaleman))
+- A memory leak in the port-forwarding of the Kubernetes dashboard and Openshift console endpoints was fixed [#4879](https://github.com/kubermatic/kubermatic/issues/4879) ([alvaroaleman](https://github.com/alvaroaleman))
+- Fixed a bug that could result to 403 errors during cluster creation when using the BringYourOwn provider [#4892](https://github.com/kubermatic/kubermatic/issues/4892) ([alvaroaleman](https://github.com/alvaroaleman))
```
Comment (xrstf) [inline CHANGELOG.md:47]: "could result **in**"

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,115 @@
...
+- ACTION REQUIRED: Added application settings view. Some of the settings were moved from config map to the `KubermaticSettings` CRD. In order to use them in the UI it is required to manually update the CRD or do it from newly added UI. [#1772](https://github.com/kubermatic/dashboard-v2/issues/1772) ([maciaszczykm](https://github.com/maciaszczykm))
+- Fixed label form validator. [#1710](https://github.com/kubermatic/dashboard-v2/issues/1710) ([maciaszczykm](https://github.com/maciaszczykm))
+- Removed `Edit Settings` option from cluster detail view and instead combine everything under `Edit Cluster`. [#1718](https://github.com/kubermatic/dashboard-v2/issues/1718) ([kgroschoff](https://github.com/kgroschoff))
+- Enabled edit options for kubeAdm [#1735](https://github.com/kubermatic/dashboard-v2/issues/1735) ([kgroschoff](https://github.com/kgroschoff))
+- Switched flag proportions to 4:3. [#1742](https://github.com/kubermatic/dashboard-v2/issues/1742) ([maciaszczykm](https://github.com/maciaszczykm))
```
Comment (xrstf) [inline CHANGELOG.md:60]: Is this really note worthy?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,115 @@
...
+- Added custom links to admin settings. [#1800](https://github.com/kubermatic/dashboard-v2/issues/1800) ([maciaszczykm](https://github.com/maciaszczykm))
+- Blocked option to edit cluster labels inherited from the project. [#1801](https://github.com/kubermatic/dashboard-v2/issues/1801) ([floreks](https://github.com/floreks))
+- Moved pod security policy configuration to the edit cluster dialog. [#1837](https://github.com/kubermatic/dashboard-v2/issues/1837) ([maciaszczykm](https://github.com/maciaszczykm))
+- Restyled some elements in the admin panel. [#1850](https://github.com/kubermatic/dashboard-v2/issues/1850) ([maciaszczykm](https://github.com/maciaszczykm))
+- Added separate save indicators for custom links in the admin panel. [#1862](https://github.com/kubermatic/dashboard-v2/issues/1862) ([maciaszczykm](https://github.com/maciaszczykm))
+
+**Addons:**
+- The dashboard addon was removed as its now deployed in the seed and can be used via its proxy endpoint [#4567](https://github.com/kubermatic/kubermatic/issues/4567) ([alvaroaleman](https://github.com/alvaroaleman))
```
Comment (xrstf) [inline CHANGELOG.md:69]: **it's** now deployed

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,115 @@
...
+- Added default namespace/cluster roles for addons [#4695](https://github.com/kubermatic/kubermatic/issues/4695) ([zreigz](https://github.com/zreigz))
+- Introduced addon configurations. [#4702](https://github.com/kubermatic/kubermatic/issues/4702) ([maciaszczykm](https://github.com/maciaszczykm))
+- Fixed addon config get and list endpoints. [#4734](https://github.com/kubermatic/kubermatic/issues/4734) ([maciaszczykm](https://github.com/maciaszczykm))
+- Added forms for addon variables. [#1846](https://github.com/kubermatic/dashboard-v2/issues/1846) ([maciaszczykm](https://github.com/maciaszczykm))
+
+**Misc:**
+- ACTION REQUIRED: Fixed missing nginx-ingress-controller permissions. Removed default publishing of services in nginx-ingress-controller. [#4688](https://github.com/kubermatic/kubermatic/issues/4688) ([xrstf](https://github.com/xrstf))
```
Comment (xrstf) [inline CHANGELOG.md:76]: The second part of this PR has been reverted later in #4990. Only the RBAC fixes are still there and they don't require any user actions during upgrades.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,115 @@
...
+- Openshift: added metrics-server [#4671](https://github.com/kubermatic/kubermatic/issues/4671) ([kron4eg](https://github.com/kron4eg))
+- For new clusters, the Kubelet port 12050 is not exposed publicly anymore [#4703](https://github.com/kubermatic/kubermatic/issues/4703) ([bashofmann](https://github.com/bashofmann))
+- The cert-manager Helm chart now creates global ClusterIssuers for Let'&#39;'s Encrypt. [#4732](https://github.com/kubermatic/kubermatic/issues/4732) ([xrstf](https://github.com/xrstf))
+- Added migration for cluster user labels [#4744](https://github.com/kubermatic/kubermatic/issues/4744) ([zreigz](https://github.com/zreigz))
+- Fixed seed-proxy controller not working in namespaces other than `kubermatic`. [#4775](https://github.com/kubermatic/kubermatic/issues/4775) ([xrstf](https://github.com/xrstf))
+- The docker logs on the nodes now get rotated via the new `logrotate` addon [#4813](https://github.com/kubermatic/kubermatic/issues/4813) ([moadqassem](https://github.com/moadqassem))
+- Made node-exporter optional addon. [#4832](https://github.com/kubermatic/kubermatic/issues/4832) ([maciaszczykm](https://github.com/maciaszczykm))
```
Comment (xrstf) [inline CHANGELOG.md:105]: **an** optional addon

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,115 @@
...
+- Added parent cluster readable name to default worker names. [#4839](https://github.com/kubermatic/kubermatic/issues/4839) ([maciaszczykm](https://github.com/maciaszczykm))
+- The QPS settings of Kubeletes can now be configured per-cluster using addon Variables [#4854](https://github.com/kubermatic/kubermatic/issues/4854) ([kdomanski](https://github.com/kdomanski))
+- Access to Kubernetes Dashboard can be now enabled/disabled by the global settings. [#4889](https://github.com/kubermatic/kubermatic/issues/4889) ([floreks](https://github.com/floreks))
+- Added support for dynamic presets [#4903](https://github.com/kubermatic/kubermatic/issues/4903) ([zreigz](https://github.com/zreigz))
+- Presets can now be filtered by datacenter [#4991](https://github.com/kubermatic/kubermatic/issues/4991) ([zreigz](https://github.com/zreigz))
+- End-of-Life Kubernetes v1.14 is no longer supported. [#4987](https://github.com/kubermatic/kubermatic/issues/4987) ([kdomanski](https://github.com/kdomanski))
```
Comment (xrstf) [inline CHANGELOG.md:111]: This should be much further up the changelog. This is waaaay more important than the n'th UI change for managing addons.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,115 @@
...
+- Restored nginx-ingress-controller --publish-service functionality. [#4990](https://github.com/kubermatic/kubermatic/issues/4990) ([xrstf](https://github.com/xrstf))
```
Comment (xrstf) [inline CHANGELOG.md:112]: We can skip this if we modify the entry above.

Comment (kdomanski) [reply CHANGELOG.md:112]: Done

File: CHANGELOG.md (hunk 8 above)
Comment (kdomanski) [reply CHANGELOG.md:111]: Done

File: CHANGELOG.md (hunk 7 above)
Comment (kdomanski) [reply CHANGELOG.md:105]: Done

File: CHANGELOG.md (hunk 6 above)
Comment (kdomanski) [reply CHANGELOG.md:76]: Done

File: CHANGELOG.md (hunk 5 above)
Comment (kdomanski) [reply CHANGELOG.md:69]: Done

File: CHANGELOG.md (hunk 4 above)
Comment (kdomanski) [reply CHANGELOG.md:60]: UI team seems to think so. @zreigz WDYT?

File: CHANGELOG.md (hunk 3 above)
Comment (kdomanski) [reply CHANGELOG.md:47]: DOne

File: CHANGELOG.md (hunk 2 above)
Comment (kdomanski) [reply CHANGELOG.md:31]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (kdomanski) [reply CHANGELOG.md:45]: Done

File: CHANGELOG.md (hunk 4 above)
Comment (zreigz) [reply CHANGELOG.md:60]: can stay

File: CHANGELOG.md [hunk 10]
```diff
@@ -21,13 +21,16 @@ Supported Kubernetes versions:
...
+- Added SUSE Linux Enterprise Server as an OS option (#659)
 
 **Cloud providers:**
 - Openstack: A bug that caused cluster reconciliation to fail if the controller crashed at the wrong time was fixed [#4754](https://github.com/kubermatic/kubermatic/issues/4754) ([alvaroaleman](https://github.com/alvaroaleman))
 - Openstack: New Kubernetes 1.16&#43; clusters use the external Cloud Controller Manager and CSI by default [#4756](https://github.com/kubermatic/kubermatic/issues/4756) ([alvaroaleman](https://github.com/alvaroaleman))
 - vSphere: Fixed a bug that resulted in a faulty cloud config when using a non-default port [#4562](https://github.com/kubermatic/kubermatic/issues/4562) ([alvaroaleman](https://github.com/alvaroaleman))
 - vSphere: fixed a bug which cased custom VM folder paths not to be put in cloud-configs [#4737](https://github.com/kubermatic/kubermatic/issues/4737) ([kdomanski](https://github.com/kdomanski))
 - vSphere: The robustness of machine reconciliation has been improved. [#4651](https://github.com/kubermatic/kubermatic/issues/4651) ([alvaroaleman](https://github.com/alvaroaleman))
+- vSphere: Sdded support for datastore clusters (#671)
```
Comment (xrstf) [inline CHANGELOG.md]: `Added`

**Pull request merged**